enclave_wrangler/objects_api.py

```diff
@@ -485,10 +487,11 @@ def fetch_cset_and_member_objects(
                 insert_fetch_statuses([{'table': 'code_sets', 'primary_key': version_id, 'status_initially':
                     'fail-excessive-members', 'comment': f"Failed after {len(cset['member_items'])} members."}])
                 call_github_action('resolve-fetch-failures-excess-items')
-        if not cset['member_items'] and cset['expression_items'] and not cset['properties']['isDraft'] and handle_issues:
+        if not cset['member_items'] and cset['expression_items'] and handle_issues:
```

Comment:
## The bugfix
If it was a draft, we knew that members were not ready yet, so we didn't bother fetching. However, there was no way to then know that the draft was complete and get the members.

This quick and simple fix does not mark as a non-draft in TermHub, but it at least gets the members. So, it partially addresses #725.

I also included a note in `fetch_audit` about the cset being a draft at the time it wasn't able to fetch members.